bin/autoindex

```diff
@@ -0,0 +1,47 @@
+
```

Comment:
Empty line to remove, otherwise we have unexpected output:

```$ php bin/autoindex 

#!/usr/bin/env php
Updating directories in /HOME/THOMAS/DOCUMENTS/GITHUB/AUTOINDEX folder ...
 142/142 [============================] 100%
```